hack/tools/go.mod

```diff
@@ -6,11 +6,11 @@ require (
 	github.com/aojea/sloppy-netparser v0.0.0-20210819225411-1b3bd8b3b975
 	github.com/cespare/prettybench v0.0.0-20150116022406-03b8cfe5406c
 	github.com/client9/misspell v0.3.4
-	github.com/golang/mock v1.4.4
-	github.com/golangci/golangci-lint v1.41.1
+	github.com/golang/mock v1.6.0
```

Comment:
this will ask you to ` hack/update-mocks.sh`

Reply:
I wondered why